docs/admin/pool_operations.md

```diff
@@ -191,174 +191,382 @@ said to be operating in "phase-2" mode.
+definitions with `bdev_roles` "meta" and "data" assigned to separate tiers:
+```bash
+    storage:
+      -
+        class: ram
+        scm_mount: /mnt/daos
+      -
+        class: nvme
+        bdev_list: ["0000:81:00.0"]
+        bdev_roles: [wal,meta]
+      -
+        class: nvme
+        bdev_list: ["0000:82:00.0"]
+        bdev_roles: [data]
+```
+
+This pool command requests to use all available storage and maintain a 1:1
+(phase-1 mode) Memory-File to Metadata-Storage:
+```bash
+$ dmg pool create bob --size 100% --mem-ratio 100%
+
+Pool created with 15.91%,84.09% storage tier ratio
+--------------------------------------------------
+  UUID             : cf70ac58-a9cd-4efd-8a96-a53697353633
+  Service Leader   : 0
+  Service Ranks    : 0
+  Storage Ranks    : 0
+  Total Size       : 951 GB
+  Metadata Storage : 151 GB (151 GB / rank)
+  Data Storage     : 800 GB (800 GB / rank)
+  Memory File Size : 151 GB (151 GB / rank)
+```
 
 Rough calculations: `dmg storage scan` shows that for each rank, one 800GB SSD
 is assigned for each tier (first: WAL+META, second: DATA). `df -h /mnt/daos*`
-reports usable ramdisk capacity for each rank is 66GiB.
-- Expected Data storage would then be 400GB for a 50% capacity first pool and
-  100% capacity second pool per-rank.
-- Expected Meta storage at 50% mem-ratio would be `66GiB*2 = 132GiB == 141GB`
-  giving ~70GB for 50% first and 100% second pools.
-- Expected Memory file size (aggregated) is `66GiB/2 = 35GB` for 50% first and
-  100% second pools.
+reports usable ramdisk capacity for the single rank is 142 GiB (152 GB).
+- Expected Data storage would then be 800GB for the pool (one rank).
+- Expected Meta storage at 100% mem-ratio would be the total ramdisk capacity.
+- Expected Memory-File size would be identical to Meta storage size.
+
+
+2. If the `--mem-ratio` is reduced to 50% in the above example, we end up with
+double the Metadata-Storage size (and a larger total pool size):
 
 ```bash
-$ dmg pool create bob --size 50% --mem-ratio 50%
+$ dmg pool create bob --size 100% --mem-ratio 50%
 
-Pool created with 14.86%,85.14% storage tier ratio
+Pool created with 27.46%,72.54% storage tier ratio
 --------------------------------------------------
-  UUID             : 47060d94-c689-4981-8c89-011beb063f8f
+  UUID             : 8e2cf446-3382-4d69-9b84-51e4e9a20c08
   Service Leader   : 0
-  Service Ranks    : [0-1]
-  Storage Ranks    : [0-1]
-  Total Size       : 940 GB
-  Metadata Storage : 140 GB (70 GB / rank)
-  Data Storage     : 800 GB (400 GB / rank)
-  Memory File Size : 70 GB (35 GB / rank)
+  Service Ranks    : 0
+  Storage Ranks    : 0
+  Total Size       : 1.1 TB
+  Metadata Storage : 303 GB (303 GB / rank)
+  Data Storage     : 800 GB (800 GB / rank)
+  Memory File Size : 151 GB (151 GB / rank)
+```
+
 
-$ dmg pool create bob2 --size 100% --mem-ratio 50%
+3. If we then try the same with 2 ranks/engines where bdev
```

Comment:
As for the first example, there is only one rank from my understanding.

Reply:
fixed